docs/cli.rst

```diff
@@ -1,11 +1,302 @@
+
+    Options
+
+        --seconds SECONDS
+        Required
+        The number of seconds to sleep
+        Default: 5
+
+Flow Info and Options
+*********************
+
+For additional information on flow ``<name>``, run either command:
+
+.. code-block:: console
+
+    $ cci flow info <name>
+    $ cci flow run --help
+
+Information about specific flows includes:
+
+* A description of the flow.
+* The ordered steps (and substeps) of a flow.
+
+..
+
+    Example: ``dev_org`` flow shows that it's composed of three subflows, ``dependencies``, ``deploy_unmanaged``, and ``config_dev``, and one task, ``snapshot_changes``. The tasks and flows making up the three subflows are also listed.
+
+.. code-block:: console
+
+    $ cci flow info dev_org
+    Description: Set up an org as a development environment for unmanaged metadata
+    1) flow: dependencies [from current folder]
+        1) task: update_dependencies
+        2) task: deploy_pre
+    2) flow: deploy_unmanaged
+        0) task: dx_convert_from
+        when: project_config.project__source_format == "sfdx" and not org_config.scratch
+        1) task: unschedule_apex
+        2) task: update_package_xml
+        when: project_config.project__source_format != "sfdx" or not org_config.scratch
+        3) task: deploy
+        when: project_config.project__source_format != "sfdx" or not org_config.scratch
+        3.1) task: dx_push
+        when: project_config.project__source_format == "sfdx" and org_config.scratch
+        4) task: uninstall_packaged_incremental
+        when: project_config.project__source_format != "sfdx" or not org_config.scratch
+    3) flow: config_dev
+        1) task: deploy_post
+        2) task: update_admin_profile
+    4) task: snapshot_changes
+
+
+
+Run Tasks and Flows
+*******************
+
+Execute a specific task or flow with the ``run`` command.
+
+.. code-block:: console
+
+    $ cci task run <name> --org <org> [options]
+    $ cci flow run <name> --org <org> [options]
+
+This command runs the respective task or flow ``<name>`` against the org ``<org>``. 
+
+.. note::
+
+    You can see a list of available orgs by running ``cci org list``.
+..
+
+    Example: The ``run_tests`` task executes Apex unit tests. Assuming there exists an org named ``dev``, you can run this task against it with the command ``cci task run run_tests --org dev``.
+
+
+
+Troubleshoot Errors
+-------------------
+
+Errors happen! That's why ``cci`` provides tools to extract error details so that they can be reported and triaged.
+
+
+Report Error Logs
+*****************
+
+The ``cci error gist`` command sends the most recent log file to a `GitHub gist <https://docs.github.com/en/github/writing-on-github/creating-gists>`_ so you can quickly and easily share logs with others. For this feature to work you need to make sure that your `GitHub  service is set up with the proper scopes <https://cumu:lusci.readthedocs.io/en/latest/tutorial.html#github-service>`_.
+
+The gist includes the:
```

Comment:
I would add the word "The" to the start of each bullet point rather than letting it hang up here.

Reply:
You're right. We end the other lists with "includes:". Three cheers for uniformity!